packaging/src/docker/6.x/.env

```diff
@@ -2,4 +2,5 @@
 ALFRESCO_EDITION=
 ALFRESCO_TAG=latest
```

Comment:
Just to note that we discussed using a fixed version of Alfresco for CI (e.g. by referencing the hash).  We could look at that at some point in the future.